config/llama2_3b_pretrain.yaml

```diff
@@ -1,27 +1,33 @@
 data: !include data/rpv1_llama.yaml
 model:
-  type: llama
+  type: llama  # Llama2-3.4B
   seq_len: 4096
-  hidden_dim: 3200
+  hidden_dim: 4096
   intermediate_dim: 8640
   num_layers: 26
   num_heads: 32
+  use_flash_attention: True
+  flash_attention_block_size: 2048
 trainer:
   wandb:
     project: "levanter"
     tags: ["redpajama", "llama"]
 
   mp: p=f32,c=bfloat16
   train_batch_size: 1024
-  per_device_parallelism: 4  # set for v3 TPU
-  per_device_eval_parallelism: 8 # set a larger batch size for eval
-  num_train_steps: 250000
+  per_device_parallelism: 8
```

Comment:
do we still need this? Does -1 work?

Reply:
This would allow us to switch the same training job between v3-64 and v3-128 seamlessly, without need to modify any parameter. I think that's quite useful especially for background run. What do you think?

Reply:
fair enough that makes sense to me